docs/magento2/search-engine.mdx

```diff
@@ -360,6 +361,45 @@ query Search {
 If you are using the default theme or the Chocolatine theme, the search bar
 should now be visible.
 
+## Attraqt recommendations
+
+> _Since version `2.19`_
+
+Front-Commerce provides a loaders to retrieve recommendations from
+[Attraqt's Widgets](https://attraqt.gitbook.io/developer-documentation/xo-recommendations/readme-first).
+
+### Front-Commerce configuration
+
+On Front-Commerce side, you need to enable the Attraqt recommendation module by
+making the following changes in your .front-commerce.js file:
+
+```js title=".front-commerce.js"
+module.exports = {
+  // highlight-next-line
+  modules: ["./node_modules/front-commerce/modules/recommendations-attraqt"],
+  serverModules: [
+    { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
+    // highlight-start
+    {
+      name: "AttraqtRecommendations",
+      path: "recommendations-attraqt/server/modules/attraqt-recommendations",
+    },
+    // highlight-end
+    { name: "Magento2", path: "server/modules/magento2" },
+  ],
+};
+```
+
+Then, in your `.env` file, you need to define the
+`FRONT_COMMERCE_ATTRAQT_RECOMMENDATION_API_URL` environment variable:
```

Comment:
**Nitpicking:** The file name (`.env`), and the environment variable name and value is also included in the code block, so repeating it here makes the docs harder to read.

```suggestion
Then, you need to define the API url in your environment variables.
```